pwa/app/routes/event.$eventKey.tsx

```diff
@@ -82,11 +84,31 @@ export default function EventPage() {
       })
     : '';
 
-  const quals = useMemo(
-    () => matches.filter((m) => m.comp_level === 'qm'),
+  const sortedMatches = useMemo(
+    () => matches.sort(sortMatchComparator),
     [matches],
   );
 
+  const quals = useMemo(
+    () => sortedMatches.filter((m) => m.comp_level === 'qm'),
+    [sortedMatches],
+  );
+
+  const elims = useMemo(
+    () => sortedMatches.filter((m) => m.comp_level !== 'qm'),
+    [sortedMatches],
+  );
+
+  const leftSideMatches =
+    quals.length > 0 ? (
+      <MatchResultsTableQuals matches={quals} />
+    ) : (
+      <MatchResultsTableDoubleElim matches={elims} />
```

Comment:
Is the eventual plan to change this based on playoff type?

Reply:
Yes